firebase-config/src/test/java/com/google/firebase/remoteconfig/FirebaseRemoteConfigTest.java

```diff
@@ -196,6 +205,7 @@ public void setUp() throws Exception {
   public void ensureInitialized_notInitialized_isNotComplete() {
     loadCacheWithConfig(mockFetchedCache, /*container=*/ null);
     loadCacheWithConfig(mockDefaultsCache, /*container=*/ null);
+    when(mockFirebaseInstanceId.getInstanceId()).thenReturn(Tasks.forResult(INSTANCE_ID_RESULT));
```

Comment:
nit: preserve order?

Reply:
I was ordering by complete then incomplete tasks, but I can use the order of tasks in `ensureInitialized` instead